components/core/src/clp/ffi/ir_stream/search/test/test_deserializer_integration.cpp

```
@@ -0,0 +1,315 @@
+            ),
+            std::make_pair(
+                    fmt::format(
+                            "@*.{} > {} AND NOT *.{} > {}",
+                            cIntKey,
+                            cIntBase,
+                            cIntKey,
+                            cIntBase
+                    ),
+                    std::vector<JsonPair>{json_pair_1, json_pair_3}
+            ),
+            std::make_pair(
```

Cppcheck (2.10-2)

[error] Syntax Error

(internalAstError)